### PR TITLE
add how to contribute

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

Fixes #<issue_number>

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -369,7 +369,8 @@ class LitAutoEncoder(pl.LightningModule):
 
 The lightning community is maintained by
 - [10+ core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, and Ph.D. students from top AI labs.
-- 400+ community contributors.
+- 480+ community active contributors.
+- read how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)
```
Comment (kaushikb11) [inline README.md:373]: Why not add the content directly in the docs instead?

Comment (Borda) [reply README.md:373]: then we can add it on both places, but many people reds just readme

**Review by awaelchli — APPROVED** [review]

**Review by akihironitta — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -79,12 +81,12 @@ A lot of good work has already been done in project mechanics (requirements.txt,
...
 
-_**Note**, even if you do not find the solution, sending a PR with a test covering the issue is a valid contribution and we can help you or finish it with you :]_
+_**Note**, even if you do not find the solution, sending a PR with a test covering the issue is a valid contribution, and we can help you or finish it with you :]_
 
 ### New Features:
 
-1. Submit a github issue - describe what is the motivation of such feature (adding the use case or an example is helpful).
-2. Let's discuss to determine the feature scope.
+1. Submit a GitHub issue - describe what is the motivation of such feature (adding the use case, or an example is helpful).
+2. Let's discuss determining the feature scope.
```
Comment (s-rog) [inline .github/CONTRIBUTING.md:89]:
```suggestion
2. Determine the feature scope with us.
```
maybe?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -230,7 +232,7 @@ We welcome any useful contribution! For your convenience here's a recommended wo
    - Make sure all tests are passing.
    - Make sure you add a GitHub issue to your PR.
 5. Use tags in PR name for following cases:
-   - **[blocked by #<number>]** if you work is depending on others changes.
+   - **[blocked by #<number>]** if your work is depending on others changes.
```
Comment (s-rog) [inline .github/CONTRIBUTING.md:235]:
```suggestion
   - **[blocked by #<number>]** if your work is dependent on other PRs.
```

File: README.md [hunk 4]
```diff
@@ -369,7 +369,8 @@ class LitAutoEncoder(pl.LightningModule):
 
 The lightning community is maintained by
 - [10+ core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, and Ph.D. students from top AI labs.
-- 400+ community contributors.
+- 480+ community active contributors.
```
Comment (s-rog) [inline README.md:372]:
```suggestion
- 480+ active community contributors.
```

File: README.md (hunk 1 above)
Comment (s-rog) [inline README.md:373]:
```suggestion
- read about how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)
```

**Review by s-rog — APPROVED** [review]
Just some wording suggestions, LGTM

**Review by SkafteNicki — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -58,13 +60,13 @@ Have a favorite feature from other libraries like fast.ai or transformers? Those
 
 ## Contribution Types
 
-We are always looking for help implementing new features or fixing bugs.
+We are always looking for help to implement new features or fixing bugs.
```
Comment (kaushikb11) [inline .github/CONTRIBUTING.md:63]:
```suggestion
We are always open to contributors helping with new features or bug fixes.
```

File: README.md [hunk 6]
```diff
@@ -369,7 +369,8 @@ class LitAutoEncoder(pl.LightningModule):
 
 The lightning community is maintained by
 - [10+ core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, and Ph.D. students from top AI labs.
-- 400+ community contributors.
+- 480+ active community contributors.
+- read about how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)
```
Comment (awaelchli) [inline README.md:373]: this can't really be a bullet point under "the lightning community is maintained by ..."

**Review by justusschock — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -58,13 +60,13 @@ Have a favorite feature from other libraries like fast.ai or transformers? Those
 
 ## Contribution Types
 
-We are always looking for help implementing new features or fixing bugs.
+We are always open to contributors helping with new features or bug fixes.
```
Comment (edenlightning) [inline .github/CONTRIBUTING.md:63]:
```suggestion
We are always open to contributions of new features or bug fixes.
```

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -2,6 +2,8 @@
 
 Welcome to the PyTorch Lightning community! We're building the most advanced research platform on the planet to implement the latest, best practices that the amazing PyTorch team rolls out!
 
+Check out our blog on how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a).
```
Comment (edenlightning) [inline .github/CONTRIBUTING.md:5]:
```suggestion
If you are new to open source, check out [this blog to get started with your first Open Source contribution](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a).
```

File: README.md [hunk 9]
```diff
@@ -369,7 +369,9 @@ class LitAutoEncoder(pl.LightningModule):
 
 The lightning community is maintained by
 - [10+ core contributors](https://pytorch-lightning.readthedocs.io/en/latest/governance.html) who are all a mix of professional engineers, Research Scientists, and Ph.D. students from top AI labs.
-- 400+ community contributors.
+- 480+ active community contributors.
+
+Want to join us? Read about how you can [start contributing](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)
```
Comment (edenlightning) [inline README.md:374]:
```suggestion
Want to help us build Lightning and reduce boilerplate for thousands of researchers? [Learn how to make your first contribution here](https://devblog.pytorchlightning.ai/quick-contribution-guide-86d977171b3a)
```

**Review by edenlightning — APPROVED** [review]

**Pull request merged**